addons/mail/static/src/owl/components/thread/thread.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="mail.component.Thread" owl="1">
         <div class="o_Thread" t-att-data-thread-local-id="props.threadLocalId">
-            <t t-if="!storeProps.threadCache or (storeProps.threadCache and (storeProps.threadCache.isLoading or !storeProps.threadCache.isLoaded))">
+            <t t-if="!storeProps.threadCache or (storeProps.threadCache and (storeProps.threadCache.isLoading or !storeProps.threadCache.isLoaded))" name="loading_condition">
```

Comment:
Just wondering if we shouldn't have a convention for naming stuffs.
Could simply be the "element" part of our flavoured BEM (Block-Element-Modifier, in our case `o_ComponentName_elementName-modifier`), so could also be camelCased.

Not sure if `xpatch` may also be applied on sub-templates, like evaluation of inner `t-call`. If that's not the case, the "element" part is enough, otherwise maybe the naming convention should also have the "block" part.

Reply:
Actually here I would like to just use the top `div` for the `xpath`, but the current syntax does not support "prepend inside", it's only "inside" (append) or "before"...

Also I'm not for a super long name here, as the `xpath` is already targeting a specific template (component) it's pointless to prefix it also on the selector.

I find the current name good enough. What would you suggest in this case?